src/prep/elaboration-core.k

```diff
@@ -182,8 +182,8 @@ when
 syntax KItem ::= haveElabRes ( KListWrap )          [function]
 rule haveElabRes([_,, elabRes(_),, _]) => true
 
-//K bug additional rule required since we introduced KListWrap
-rule haveElabRes([_,, KList2KLabel _,, elabRes(_),, _ (_),, _]) => true
+//K bug additional rule required since we introduced KListWrap? todo (Shijiao)
+//rule haveElabRes([_,, KList2KLabel _,, elabRes(_),, _ (_),, _]) => true
```

Comment:
I don't really understand this rule. KList2KLabel is not occuring anywhere else.
I got the following error when uncommenting it:
[Error] Critical: Could not infer a unique sort for variable 'KList2KLabel'.
Possible sorts: AnnoElemDec, SwitchLabel, Set, ClassDecHead, AnnoFieldMod,
TypeDecSpec, StringPart, SwitchGroup, InterfaceMemberDec, AnnoMethodMod,
SwitchLabelList, EnumDecHead, Anno, AbstractMethodMod, DimExp, TypeDec, List,
AnnoConstrMod, ConstrHead, AnnoClassMod, AnnoConstantMod, AnnoVarMod,
BlockStmt, Dim, CatchClause, MethodDecHead, Type, AnnoAbstractMethodMod,
TypeVarId, ClassBodyDec, Id, AnnoInterfaceMod, Map, ImportDec
	Source: File: /home/yuwen/java-semantics/src/prep/elaboration-core.k
	Location: (186,1,186,71)